roles/kubernetes/node/templates/kubelet.env.v1beta1.j2

```diff
@@ -11,7 +11,9 @@ KUBELET_HOSTNAME="--hostname-override={{ kube_override_hostname }}"
 --config={{ kube_config_dir }}/kubelet-config.yaml \
 --kubeconfig={{ kube_config_dir }}/kubelet.conf \
 {# end kubeadm specific settings #}
+{% if kube_version is version('1.27', '<') %}
 --container-runtime=remote \
```

Comment:
IIUC removing this regardless of the version should be fine, see this: https://github.com/kubernetes/website/commit/68ca0560f08a359bcfd24dcdb50a49247c13278d#diff-a7a37ad9496edf0e4b1ef0cf4270571e43d83d29a9b8fd9c14b6a155a6746e38L199

Reply:
I tried to install k8s on clean servers several times and got the error:

```
May 27 22:47:14 node1 kubelet[22748]: E0527 22:47:14.341965   22748 run.go:74] "command failed" err="failed to parse kubelet flag: unknown flag: --container-runtime"
May 27 22:47:14 node1 systemd[1]: kubelet.service: Main process exited, code=exited, status=1/FAILURE
```

Reply:
Yes I meant removing the argument in any version like without checking if the version is at least 1.27

Reply:
done